src/base/dev/vga/vgaemu.c

```diff
@@ -1750,13 +1750,7 @@ int vga_emu_pre_init(void)
       /* create mapping:
 	 Linear vga.mem.lfb_base -> phys VGAEMU_PHYS_LFB_BASE=0xe0000000 ->
 	 addr (hugepage aligned) -> vga.mem.base */
```

Comment:
This comment is now out of date. I suggest removing it.

Reply:
Done.